src/main/java/cart/controller/admin/AdminProductController.java

```diff
@@ -0,0 +1,45 @@
+package cart.controller.admin;
+
+import cart.controller.dto.ProductDto;
+import cart.service.ProductService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.Valid;
+import java.net.URI;
+
+@RestController
+@RequestMapping("/admin/products")
+public class AdminProductController {
+
+    private final ProductService productService;
+
+    public AdminProductController(final ProductService productService) {
+        this.productService = productService;
+    }
+
+    @PostMapping
+    public ResponseEntity<Void> addProduct(@RequestBody @Valid final ProductDto productDto) {
+        final Long productId = productService.save(productDto);
+        return ResponseEntity.created(URI.create("/admin/" + productId)).build();
```

Comment:
Location 값이 잘못된 듯 해요!

```suggestion
        return ResponseEntity.created(URI.create("/admin/product/" + productId)).build();
```